test/unittests/execute_numeric_test.cpp

```diff
@@ -49,7 +49,21 @@ TypedExecutionResult execute_binary_operation(Instr instr, TypedValue lhs, Typed
         {static_cast<uint8_t>(Instr::local_get), 0, 0, 0, 0, static_cast<uint8_t>(Instr::local_get),
             1, 0, 0, 0, static_cast<uint8_t>(instr), static_cast<uint8_t>(Instr::end)}});
 
-    return execute(*instantiate(std::move(module)), 0, {lhs, rhs});
+    auto instance = instantiate(std::move(module));
+
+    return [instance = std::move(instance)](TypedValue lhs, TypedValue rhs) {
+        return execute(*instance, 0, {lhs, rhs});
+    };
+}
+
+TypedExecutionResult execute_unary_operation(Instr instr, TypedValue arg)
+{
+    return create_unary_operation_executor(instr)(arg);
```

Comment:
Will these create helpers used later in other PRs?

Reply:
The `i64_eqz` is using it.

Reply:
The point is to avoid type check or why is it needed?

Reply:
It is not needed, but it is lame to create module+instance for every test case. Here we create a lambda capturing the instance and we can execute multiple test cases.

Reply:
Well, the fresh instance gives the benefit, that nothing is dirty anywhere in the instance. However this way perhaps we could catch other bugs/problems caused by some uncleaned state.

Reply:
> It is not needed, but it is lame to create module+instance for every test case. Here we create a lambda capturing the instance and we can execute multiple test cases.

But there are many tests still using `execute_unary_operation` / `execute_binary_operation`, why leave it inconsistent?

Reply:
This PR mostly enables you to do the change. I needed it for more `eqz` test cases. For other instructions I prefer to go one-by-one later and add more test cases (something more than `42`). But I can syntactically replace the code right now if you prefer all at once (will not add more cases though).

Reply:
I would prefer to replace all of them here, if it's easy.